store/migration/dev/LATEST.sql

```diff
@@ -567,10 +567,13 @@ CREATE TABLE backup_setting (
     updater_id INTEGER NOT NULL REFERENCES principal (id),
     updated_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
     database_id INTEGER NOT NULL REFERENCES db (id),
+    -- enable automatic backup schedule.
     enabled BOOLEAN NOT NULL,
     hour INTEGER NOT NULL CHECK (hour >= 0 AND hour <= 23),
     -- day_of_week can be -1 which is wildcard (daily automatic backup).
     day_of_week INTEGER NOT NULL CHECK (day_of_week >= -1 AND day_of_week <= 6),
+    -- retention_period_ts == -1 means infinite retention period, 0 means do not keep any data.
```

Comment:
It's a bit strange to see -1. Is it possible to use Nullable?

Reply:
SG.